lib/views/pages/login_signup/register_form.dart

```diff
@@ -471,7 +465,8 @@ class RegisterFormState extends State<RegisterForm> {
           );
         });
   }
-  /*_successToast(String msg) {
+
```

Comment:
This changes are already added in PR https://github.com/PalisadoesFoundation/talawa/pull/373

Reply:
When I made the PR, they weren't and that's why I did them. No problem, I'll just revert these changes, pull and push.

Reply:
@Sagar2366 , I've made the required changes. Kindly review them.